src/components/side-panel/side-panel.component.ts

```diff
@@ -107,16 +104,18 @@ export class SidePanelComponent implements OnInit, OnDestroy {
         private readonly _focusOrigin: FocusIndicatorOriginService) { }
 
     ngOnInit(): void {
+
+        this.service.open$.pipe(skip(1), distinctUntilChanged(), takeUntil(this._onDestroy))
+            .subscribe(isOpen => this.openChange.emit(isOpen));
+
         this.service.open$.pipe(distinctUntilChanged(), takeUntil(this._onDestroy)).subscribe(isOpen => {
             this.animationPanelState = isOpen
                 ? this.animate
                     ? SidePanelAnimationState.Open
                     : SidePanelAnimationState.OpenImmediate
                 : SidePanelAnimationState.Closed;
 
-            // only if the open state changed should we emit the latest value
             this.openChange.emit(isOpen);
```

Comment:
I think this should be removed as it is handled by the other subscription?

Reply:
removed